### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by BeataSolitudo**

### CODE REVIEW

**Comment by BeataSolitudo** [comment]
Update readme

File: README.md [hunk 1]
```diff
@@ -55,10 +55,10 @@ To run this project in your development machine, follow these steps:
 1. (optional) Create and activate a [virtualenv](https://virtualenv.pypa.io/) (you may want to use [virtualenvwrapper](http://virtualenvwrapper.readthedocs.org/)).
 
 2. Ensure that the executable `pg_config` is available on your machine. You can check this using `which pg_config`. Otherwise, sqlite will be used.
+ 
+**Note:** pg_config is not needed. You can use sqlite instead.
 
-3. Fork this repo and clone your fork:
-
-    `git clone https://github.com/sclorg/django-ex.git`
+3. Fork this repo and clone your fork: Fork this repo and clone your fork. Make sure that you are in the right branch.
```
Comment (frenzymadness) [inline README.md:61]: The first sentence is twice there.

**Review by frenzymadness — CHANGES_REQUESTED** [review]
I've found one typo. Could you please fix it?

**Review by frenzymadness — APPROVED** [review]
Looks good to me.

**Pull request merged**